core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -48,6 +48,12 @@
     @ConfigItem(defaultValue = "false")
     public boolean addAllCharsets;
 
+    /**
+     * If all timezones sets should be added to the native image. This increases image size
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean addAllTimeZones;
+
```

Comment:
Hello @machi1990, does this mean the user will have to use `quarkus.native.add-all-time-zones` as the property in the `application.properties`?  If so, can we maybe just call it `enableAllTimezones`, so that they can consistently use `quarkus.native.enable-all-timezones`?

Reply:
Hi @jaikiran thanks for the quick review. 

> Hello @machi1990, does this mean the user will have to use `quarkus.native.add-all-time-zones` as the property in the `application.properties`? 

Yes

If so, can we maybe just call it `enableAllTimezones`, so that they can consistently use `quarkus.native.enable-all-timezones`?

+1. let me do the naming change.

I think
```
@ConfigItem(defaultValue = "false", name="nable-all-timezones")
public boolean addAllTimeZones;
```
Will do the job :-)

Reply:
Well, `addAllCharsets` and `addAllTimeZones` should be consistent because they do the same thing.

I agree that having the build items and the config consistent would be nice. I don't have a strong opinion about keeping either.

@Sanne WDYT about what would be the best name?

Reply:
I'm not sure to expose this configuration attribute.

As I explained on the mailing list, there's many good reasons to _not_ have it here. I might be wrong, or I might be persuaded otherwise, but we could also simply start without having it and then add this later when there's better elements to decide.

Always easier to add things later than to remove it.

Reply:
P.S. actually I'm fairly sure we shouldn't expose this yet: at the very least we'll want to expose the option listing individual timezones, but we need to wait to be based on a GraalVM version which includes @jaikiran 's fixes

Reply:
+1 avec Sanne. I’ll remove the config.

Reply:
I have removed the config. We can expose it to users later based on @jaikiran GraalVM fixes.